src/mltk/predictor/gam/GA2MLearner.java

```diff
@@ -118,9 +124,13 @@ public static void main(String[] args) throws Exception {
 
 		Instances trainSet = InstancesReader.read(opts.attPath, opts.trainPath);
 
+		int numPairs = (int) opts.numPairs;
```

Comment:
The user needs to provide a file of all pairs of features that he/she wants to use, it seems that the complexity of usability is increased when one needs to think of two parameters (ranking of all pairs and how many pairs to use).